client/modules/IDE/components/Editor.jsx

```diff
@@ -62,7 +62,11 @@ const beautifyHTML = beautifyJS.html;
 window.JSHINT = JSHINT;
 window.CSSLint = CSSLint;
 window.HTMLHint = HTMLHint;
-delete CodeMirror.keyMap.sublime['Shift-Tab'];
+
+
+// delete CodeMirror.keyMap.sublime['Shift-Cmd-F'];
+
+console.log(CodeMirror.keyMap.sublime);
```

Comment:
Could you delete this `console.log` and the commented out code?

Reply:
Yes!